legal-api/src/legal_api/models/filing.py

```diff
@@ -198,7 +198,11 @@ class Source(Enum):
                     'BEN': 'BCFDR',
                     'BC': 'BCFDR',
                     'ULC': 'BCFDR',
-                    'CC': 'BCFDR'
+                    'CC': 'BCFDR',
+                    'CBEN': 'BCFDR',
+                    'C': 'BCFDR',
+                    'CUL': 'BCFDR',
+                    'CCC': 'BCFDR'
```

Comment:
👍 

Can you please link this PR to the ticket?

Reply:
Or is it not related to this ticket?

Reply:
not related to that bug. don't have a ticket for this

Reply:
OK. Resolving.